src/components/test/ShortcutsModal.spec.js

```diff
@@ -6,15 +6,34 @@ import React from "react";
 import { shallow } from "enzyme";
 import { ShortcutsModal } from "../ShortcutsModal";
 
+function render(overrides = {}) {
```

Comment:
Love this rewrite, so sophisticated :)

Reply:
not going to claim this. Learned this pattern from other files from this project.